pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -267,12 +267,11 @@ func classifyVolumesForMigration(vmi *v1.VirtualMachineInstance) *migrationDisks
 				disks.shared[volume.Name] = true
 			}
 		case volSrc.HostDisk != nil:
-			if volSrc.HostDisk.Shared != nil && *volSrc.HostDisk.Shared {
-				disks.shared[volume.Name] = true
-			} else if _, ok := migrateDisks[volume.Name]; ok {
+			if _, ok := migrateDisks[volume.Name]; ok {
```

Comment:
just a nitpick, definitely not for this PR
normally in go you'd use an empty struct instead of the bool in this kind of maps, to spare the allocation:
https://stackoverflow.com/a/22770744